### PR TITLE
54 list user emails

### LINKED ISSUE #54: Admin: View list of new user emails for adding to flowdock
As a Code for Denver admin onboarding new members, I would like an easy way to invite new members to Flowdock.

I believe the simplest way of achieving this initially is to put a list show a comma-separated list of emails of users (preferably filtered to be just those that signed up in the last week).

### PR DESCRIPTION
**Pull request by jksmall0631**
This PR closes #54 

#### What does this PR do?
-Modified members component to list all user emails

### CODE REVIEW

File: src/Members.js [hunk 1]
```diff
@@ -1,34 +0,0 @@
-import React from "react";
-import gql from "graphql-tag";
-import { graphql } from "react-apollo";
-
-const Members = ({ members }) => {
-  return (
-    <div>
-      {members.map(member =>
-        <div key={member.email}>
-          {member.email}
```
Comment (wdoug) [inline src/Members.js:10]: Interesting, we were already displaying users emails? Do we have other data for users such as their name?

File: src/UsersList.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  );
+};
+
+UsersList.defaultProps = {
+  users: []
+};
+
+const allUsersQuery = gql`
+  query users {
+    allUsers {
```
Comment (wdoug) [inline src/UsersList.js:23]: Is there a way to filter by when users joined? Do we even store that data?

File: src/App.js [hunk 3]
```diff
@@ -17,36 +17,35 @@ class App extends Component {
...
           exact path="/me"
-          render={() =>
-            <MemberProfile
-              user={this.props.data.user}
-            />}
+          render={() => <MemberProfile user={user} />}
         />
         <Route path="/me/edit" component={MemberProfileEdit} />
-      </div>
+        <Route path="/user-list" component={UsersList} />      </div>
```
Comment (wdoug) [inline src/App.js]: For this story, being a list of emails for the admin to add people, I think the route should be something like `/admin/onboarding`, and then card #55 could be the user list that would display more public info, like name, etc.

File: src/Members.js (hunk 1 above)
Comment (jksmall0631) [reply src/Members.js:10]: We were displaying Members before which was a seperate table in graph.cool. I just had to switch up the query. We can display name and email if you want, sure.

File: src/UsersList.js (hunk 2 above)
Comment (jksmall0631) [reply src/UsersList.js:23]: Yeah, we have a createdAt value. Looks like graph.cool will allow us to sort by this date or we can just create a query to grab only the people who signed up on that day. Let me know which you would like to do. @wdoug

File: src/App.js (hunk 3 above)
Comment (jksmall0631) [reply src/App.js]: That sounds good to me, I'll make those changes now.

**Review by dviramontes — APPROVED** [review]
👍

**Comment by dviramontes** [comment]
Made separate issues for the outstanding questions/features.

**Pull request merged**